packages/dev/core/src/FlowGraph/Blocks/Data/flowGraphBinaryOpBlock.ts

```diff
@@ -5,7 +5,16 @@ import type { FlowGraphDataConnection } from "../../flowGraphDataConnection";
 /**
  * @experimental
  */
-class FlowGraphBinaryOpBaseBlock<LeftT, RightT, OutputT> extends FlowGraphBlock {
+export interface IFlowGraphBinaryOpBaseBlockParams<LeftT, RightT, OutputT> {
+    defaultLeftValue: LeftT;

```

Comment:
I tend to agree with @RaananW overall, value should always be optional with a default value (when it makes sense)